### PR TITLE
Make Stupidchacha.cpp

### PR DESCRIPTION
**Pull request by royeom**
chaminjun is stupid

### CODE REVIEW

File: stupidchacha.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main()
+{
+	std::cout<<"Stupid chaminjun"<<std::endl;
+}
```
Comment (mjkoo711) [inline stupidchacha.cpp]: You don't have return!! You are stupid!!!

**Review by mjkoo711 — COMMENTED** [review]
check!

File: stupidchacha.cpp (hunk 1 above)
Comment (royeom) [reply stupidchacha.cpp]: ok ok

Comment (mjkoo711) [reply stupidchacha.cpp]: it's good!!

**Review by mjkoo711 — APPROVED** [review]
LGTM

**Pull request merged**